ext/opentelemetry-ext-sqlalchemy/tests/test_sqlalchemy.py

```diff
@@ -0,0 +1,48 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from unittest import mock
+
+from sqlalchemy import create_engine
+
+from opentelemetry import trace as trace_api
+from opentelemetry.ext.sqlalchemy.engine import trace_engine
+
+
+class TestSqlalchemyInstrumentation(unittest.TestCase):
```

Comment:
What about using TestBase here and avoiding the mocking?

Reply:
Good point, cleans up the test quite a bit.